### DISCUSSION TITLE
[Support]: Snapshot coordinates mismatch

### DISCUSSION THREAD

**Discussion by esvaf** [blakeblackshear/frigate#12149] [Report a Bug] [answered]
### Describe the problem you are having

I am trying to crop event snapshots by region, but the outlined actual object often goes beyond the designated area. It seems that the coordinates provided do not correspond to the time the snapshot was taken, and I suspect this might be a bug. Below, I have included an example of the full snapshot and the cropped version according to the given coordinates. I am also attaching the coordinates from the API.
![snapshot](https://github.com/blakeblackshear/frigate/assets/17836561/701a3819-2c3a-4710-ba33-e1a11dd712d5)
![snapshot](https://github.com/blakeblackshear/frigate/assets/17836561/93a4d5f0-535f-487d-b63b-4b4a1277a92d)

{
  "area": null,
  "box": null,
  "camera": "camname",
  "data": {
    "attributes": [],
    "box": [
      0.16463414634146342,
      0.3840277777777778,
      0.29796747967479675,
      0.6104166666666667
    ],
    "region": [
      0.08495934959349594,
      0.28055555555555556,
      0.4211382113821138,
      0.7194444444444444
    ],
    "score": 0.96484375,
    "top_score": 0.98828125,
    "type": "object"
  },
  "detector_type": "edgetpu",
  "end_time": 1719259084.195377,
  "false_positive": null,
  "has_clip": true,
  "has_snapshot": true,
  "id": "1719259067.956302-0asycd",
  "label": "car",
  "model_hash": "1fa745af0539ba5a2e001ea60d470584",
  "model_type": "ssd",
  "plus_id": null,
  "ratio": 1,
  "region": null,
  "retain_indefinitely": false,
  "score": null,
  "start_time": 1719259062.956302,
  "sub_label": null,
  "thumbnail": "< omitted >",
  "top_score": null,
  "zones": [
    "road_area"
  ]
}

### Steps to reproduce

Wait for a fast-moving object and check the snapshot.

### Version

0.13.2

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
NA
```

### Operating system

Other Linux

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Hikvision

**Comment by NickM-27** [comment]
So to be clear which coordinates are you using to crop?

**Reply by esvaf** [reply]
> So to be clear which coordinates are you using to crop?

I use the coordinates of the region because they cover a slightly larger area of the frame than the coordinates of the box.

**Reply by NickM-27** [reply, marked as answer]
Okay, this is not a bug I think you are just using the coordinates wrong. The coordinates are `[x, y, width, height]` so it is impossible for the box to be outside of the region because in this case the right edge of the region is 0.5 while the right edge of the box is 0.45

**Reply by esvaf** [reply]
> The coordinates are `[x, y, width, height]` 
I couldn't find a description of these values anywhere in the documentation and was sure that these were the coordinates of the corners. Thanks a lot! Everything looks correct now.
![snapshot](https://github.com/blakeblackshear/frigate/assets/17836561/14ccd81d-9485-4c0b-b72e-53c824081185)
